### DISCUSSION TITLE
video player closed with error

### DISCUSSION THREAD

**Discussion by Yefita** [pystardust/ani-cli#548] [Q&A] [answered]
I have ani-cli installed on my android for a while.  everything work great until yesterday, I factory reset my phone and install the script again.

when I run the script, it run like normal. but as mpv open, it close almost instantly. when I use vlc it also closed with this popup.

![IMG_20220220_225413](https://user-images.githubusercontent.com/99162380/154852162-0a6164a8-18af-4c35-9f87-82f6f7452ec9.jpg)

How do I fix this? Ive try to reinstall it but no luck

**Comment by RaynardGerraldo** [comment, marked as answer]
> I have ani-cli installed on my android for a while. everything work great until yesterday, I factory reset my phone and install the script again.
> 
> when I run the script, it run like normal. but as mpv open, it close almost instantly. when I use vlc it also closed with this popup.
> 
> ![IMG_20220220_225413](https://user-images.githubusercontent.com/99162380/154852162-0a6164a8-18af-4c35-9f87-82f6f7452ec9.jpg)
> 
> How do I fix this? Ive try to reinstall it but no luck

This is due to the issue in #543,a solution to this would be using mpv and editing the mpv.conf file

> Open mpv, goto three dots top right->Settings->Advanced-->Edit mpv.conf

Then add `referrer="https://gogoanime.fi"`

**Reply by Yefita** [reply]
it did fix it. thanks
